packages/auth-server/components/app/nav.vue

```diff
@@ -80,6 +85,13 @@ onBeforeUnmount(() => {
 });
 
 watch(windowWidth, checkWidths);
+
+const { logout: _logout } = useAccountStore();
```

Comment:
The underscore makes mething that it's not used. But it's actually to avoid name collision. What do you think about rename any of them instead of using underscore?